scripts/utils/fcp_pop_helper.py

```diff
@@ -42,8 +44,10 @@ def get_fcp_population(args,
         ]
 
         if parallel:
-            with multiprocessing.Pool() as pool:
-                dilled_results = pool.starmap(train_agent_with_checkpoints, inputs)
+            with concurrent.futures.ProcessPoolExecutor(max_workers=max_concurrent_jobs) as executor:
```

Comment:
Thanks! Can you put max_concurrent_jobs in `arguments.py` so instead of passing it to all the functions, we can just call `max_workers=args.max_concurrent_jobs`?